src/services/NodeService.jsx

```diff
@@ -1,18 +1,7 @@
 import lokiService from './LokiService';
 import ISO8601ServiceInstance from './ISO8601Service';
 
-/**
- * @class NodeService
```

Comment:
Why remove description comments completely? These definitely need to stay

Reply:
The descriptions didn't feel accurate so I was just removing them for now. Like the NodeService here does more than just add new nodes. I like function descriptions since it's easier to summarize but the class level descriptions don't feel like they add much value. I can leave them in if you want to keep them though.